### PR TITLE
Fix return values for attack, Initialize, and Destroy procs

### PR DESCRIPTION
**Pull request by out-of-phaze**
## Description of changes
Fixes all the incorrect return values for attackby/attack_hand, Initialize(), and Destroy() procs.

I am so sorry, pre-emptively, to whoever reviews this

## Why and what will this PR improve
Everything should return `TRUE` or `FALSE` now, barring a few edge-cases where things return the value of a proc and I didn't want to go and audit all of THOSE too.

## Authorship
Me me me, including the OpenDream stuff needed to find these

### CODE REVIEW

File: code/game/machinery/computer/law.dm [hunk 1]
```diff
@@ -8,8 +8,9 @@
 	if(istype(O, /obj/item/aiModule))
 		var/obj/item/aiModule/M = O
 		M.install(src, user)
+		return TRUE
 	else
```
Comment (MistakeNot4892) [inline code/game/machinery/computer/law.dm:12]: Would suggest removing the else and deindenting the return, but not blocking.

File: code/game/machinery/embedded_controller/airlock_docking_controller.dm [hunk 2]
```diff
@@ -21,8 +21,9 @@
 		else
 			code = stars(code)
 		to_chat(user,"[W]'s screen displays '[code]'")
+		return TRUE
 	else
```
Comment (MistakeNot4892) [inline code/game/machinery/embedded_controller/airlock_docking_controller.dm:25]: Same as prior comment.

File: code/game/machinery/flasher.dm [hunk 3]
```diff
@@ -43,8 +43,9 @@
 			user.visible_message("<span class='warning'>[user] has disconnected \the [src]'s flashbulb!</span>", "<span class='warning'>You disconnect \the [src]'s flashbulb!</span>")
 		if (!src.disable)
 			user.visible_message("<span class='warning'>[user] has connected \the [src]'s flashbulb!</span>", "<span class='warning'>You connect \the [src]'s flashbulb!</span>")
+		return TRUE
 	else
```
Comment (MistakeNot4892) [inline code/game/machinery/flasher.dm:47]: Same as prior comment.

File: code/game/machinery/igniter.dm [hunk 4]
```diff
@@ -128,8 +128,9 @@
 		else if(!disable)
 			user.visible_message("<span class='warning'>[user] has reconnected \the [src]!</span>", "<span class='warning'>You fix the connection to \the [src].</span>")
 		update_icon()
+		return TRUE
 	else
```
Comment (MistakeNot4892) [inline code/game/machinery/igniter.dm:132]: Same as prior.

File: code/game/machinery/kitchen/icecream.dm [hunk 5]
```diff
@@ -132,11 +132,11 @@
 				to_chat(user, "<span class='warning'>There is not enough icecream left!</span>")
 		else
 			to_chat(user, "<span class='notice'>[O] already has icecream in it.</span>")
-		return 1
+		return TRUE
 	else if(ATOM_IS_OPEN_CONTAINER(O))
-		return
+		return TRUE
 	else
```
Comment (MistakeNot4892) [inline code/game/machinery/kitchen/icecream.dm:138]: Same as prior.

File: code/game/machinery/kitchen/smartfridge.dm [hunk 6]
```diff
@@ -187,7 +187,7 @@
 /obj/machinery/smartfridge/attackby(var/obj/item/O, var/mob/user)
 	if(accept_check(O))
 		if(!user.try_unequip(O))
```
Comment (MistakeNot4892) [inline code/game/machinery/kitchen/smartfridge.dm:189]: Could invert this and indent the block below to remove an extraneous return, not blocking.

File: code/game/machinery/oxygen_pump.dm [hunk 7]
```diff
@@ -143,18 +144,21 @@
...
+		if(!user.try_unequip(W, src))
+			return TRUE
+		tank = W
+		user.visible_message(SPAN_NOTICE("\The [user] installs \the [tank] into \the [src]."), SPAN_NOTICE("You install \the [tank] into \the [src]."))
+		src.add_fingerprint(user)
+		return TRUE
 	if(istype(W, /obj/item/tank) && !stat)
 		to_chat(user, SPAN_WARNING("Please open the maintenance hatch first."))
+		return TRUE
+	return FALSE // TODO: should this be a parent call? do we want this to be (de)constructable?
```
Comment (MistakeNot4892) [inline code/game/machinery/oxygen_pump.dm:161]: I think in theory we want all machinery to be deconstructable?

File: code/game/machinery/portable_turret.dm [hunk 8]
```diff
@@ -236,6 +236,7 @@ var/global/list/turret_icons
 		new /obj/item/assembly/prox_sensor(loc)
 	. = ..()
 
+// TODO: remove these or refactor to use construct states
```
Comment (MistakeNot4892) [inline code/game/machinery/portable_turret.dm:239]: We have nuturrets, I'm not actually sure why we still have these porta turrets, they're atrocious.

File: code/game/machinery/computer/law.dm (hunk 1 above)
Comment (out-of-phaze) [reply code/game/machinery/computer/law.dm:12]: ahaha, i wanted to do it for all of these but figured I should leave it alone bc i think i was told to do that prior. more than happy to but also keeping them in served an important purpose originally (making sure typemaker can catch them)

Comment (MistakeNot4892) [reply code/game/machinery/computer/law.dm:12]: Up to you, I am not particularly fussed, just a tiny bit cleaner/more resilient to people adding earlier conditions.

File: code/game/machinery/requests_console.dm [hunk 9]
```diff
@@ -208,26 +208,29 @@ var/global/req_console_information = list()
...
-		if(screen == RCS_ANNOUNCE)
-			var/obj/item/card/id/ID = O
-			if (access_RC_announce in ID.GetAccess())
-				announceAuth = 1
-				announcement.announcer = ID.assignment ? "[ID.assignment] [ID.registered_name]" : ID.registered_name
-			else
-				reset_message()
-				to_chat(user, "<span class='warning'>You are not authorized to send announcements.</span>")
-			SSnano.update_uis(src)
+		if(inoperable(MAINT)) return TRUE
```
Comment (MistakeNot4892) [inline code/game/machinery/requests_console.dm:211]: As a minor stylistic note, I think we probably want returns on their own lines.

File: code/game/objects/structures/flaps.dm [hunk 10]
```diff
@@ -47,11 +47,14 @@
...
 			user.visible_message("<span class='warning'>\The [user] deconstructs \the [src].</span>", "<span class='warning'>You deconstruct \the [src].</span>")
 			qdel(src)
+		return TRUE
 	if(IS_SCREWDRIVER(W) && anchored)
 		airtight = !airtight
 		airtight ? become_airtight() : clear_airtight()
 		user.visible_message("<span class='warning'>\The [user] adjusts \the [src], [airtight ? "preventing" : "allowing"] air flow.</span>")
-	else ..()
+		return TRUE
+	else
```
Comment (MistakeNot4892) [inline code/game/objects/structures/flaps.dm:56]: Same as above.

File: code/game/objects/items/weapons/policetape.dm [hunk 11]
```diff
@@ -89,7 +89,8 @@ var/global/list/image/hazard_overlays //Cached hazard floor overlays for the bar
 	return ..()
 
 /obj/item/stack/tape_roll/barricade_tape/attack_hand()
-	if((. = ..()) && !QDELETED(src))
```
Comment (MistakeNot4892) [inline code/game/objects/items/weapons/policetape.dm:92]: whyfor

File: code/modules/paperwork/paper_bundle.dm [hunk 12]
```diff
@@ -54,7 +54,8 @@
 			. = P.attackby(W, user)
 			update_icon()
 			updateUsrDialog()
-		return
+			return
+		// How did we not have a page? Dunno, fall through to parent call anyway, I guess
```
Comment (MistakeNot4892) [inline code/modules/paperwork/paper_bundle.dm:58]: Copypasted comment?

**Review by MistakeNot4892 — DISMISSED** [review]
This was not a thorough review but I did look at every file

**Comment by out-of-phaze** [comment]
that force-push was conflict resolution, it's been approved for 25 hours now

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**